docs/wiki/research/PBS/PTC-Specs.md

```diff
@@ -0,0 +1,484 @@
+# ePBS Design Specifications
```

Comment:
the doc name suggest PTC specs, I guess it's meant to be `ePBS-specs.md`?

Reply:
Well, it is a ePBS solution based on PTC scheme. Since there was already a detailed proposals for ePBS based on PEPC, we may see ePBS-PEPC-specs sometime in the future. I will rename it to `ePBS-PTC-Specs.md` for now.